includes/Dashboard.php

```diff
@@ -165,12 +165,17 @@ public function load_stories_dashboard() {
 		 */
 		$preload_paths = apply_filters( 'web_stories_dashboard_preload_paths', $preload_paths );
 
+		$_GET['_embed'] = 1;
```

Comment:
```suggestion
		$_GET['_embed'] = 'author';
```

Shouldn't it be this?

Reply:
Hmm true.

Although to be 100% sure it would actually need to be set per path, see https://xwp.slack.com/archives/C013J2EGH8R/p1601567869057300?thread_ts=1601567500.056700&cid=C013J2EGH8R